### PR TITLE
Article added

### PR DESCRIPTION
**Pull request by oshastitko**
Added ability to get article list by search word

### CODE REVIEW

File: src/ZendeskApi.Client/Resources/Article/ArticlesResources.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+using System.Text;
+using Microsoft.Extensions.Logging;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Resources.Interfaces;
+using ZendeskApi.Client.Extensions;
+using ZendeskApi.Client.Exceptions;
+using ZendeskApi.Client.Responses.Articles;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources.Articles
```
Comment (LoadData) [inline src/ZendeskApi.Client/Resources/Article/ArticlesResources.cs]: Could we keep this under the `ZendeskApi.Client.Resources` namespace?

File: src/ZendeskApi.Client/Resources/Article/ArticlesResources.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        private readonly Func<ILogger, string, IDisposable> _loggerScope = LoggerMessage.DefineScope<string>(typeof(ArticlesResources).Name + ": {0}");
+
+        public ArticlesResources(IZendeskApiClient apiClient, ILogger logger)
+        {
+            _apiClient = apiClient;
+            _logger = logger;
+        }
+
+
+        public async System.Threading.Tasks.Task<Responses.IPagination<Article>> ListAsync(string query, PagerParameters pager = null)
```
Comment (LoadData) [inline src/ZendeskApi.Client/Resources/Article/ArticlesResources.cs]: Can we add the `using`'s for this instead, and just have it as `async Task<IPagination<Article>>`.

Would you mind also changing it's name to 'GetAsync'? This is the new convention we're trying to adhere to, and we're slowly rolling it out throughout resources.

**Comment by jonathanstowell** [comment]
I think this will need to be refactored now to use the new AbstractBaseResource which helps push the conventions through each resource. We could also do with some Tests around it. See the ZendeskApi.Client.Tests project for that. Ignore the IntegrationTests project.

**Comment by jonathanstowell** [comment]
Articles were added a little while ago (July):

https://github.com/justeat/ZendeskApiClient/blob/master/src/ZendeskApi.Client/Resources/HelpCenter/HelpCenterArticlesResource.cs#L10

Going to close this PR.

**Pull request closed without merging**